taxi/models.py

```diff
@@ -0,0 +1,37 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    def __str__(self) -> str:
+        return f"{self.name} {self.country}"
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=63, unique=True)
+
+    class Meta:
+        ordering = ('username', )
+
+    def __str__(self) -> str:
+        return f"{self.username} ({self.first_name} {self.last_name})"

```

Comment:
Similar to the previous comment, avoid overusing f-strings. The `__str__` method for the `Driver` class can return the username directly if that's sufficient for your needs.